app/templates/main/index.html

```diff
@@ -40,11 +40,41 @@ <h3 id="stock_graph">Alphabet Inc</h3>
           </div>
         </div>
         <div class="card-footer">
-          <strong>
-            This area here will give more details of the stock current values.<br>
-          </strong>
-          Lorem, ipsum dolor sit amet consectetur adipisicing elit. Ab eius ex nihil quam nesciunt architecto corporis fugit, cupiditate hic quidem sapiente, quia mollitia vero dolor harum praesentium quas officiis accusantium! <br>
-          <a href="#">A link to the stock details page</a>
+        <head>
+          <h3>Stock Values</h3>
+          <style>table, th, td{ border: 1px solid black; }</style>
+        </head>
+          <table>
+          <thead>
+          <tr>
+            <th>Open</th>
+            <th>High</th>
+            <th>Low</th>
+            <th>Mkt cap</th>
+            <th>P/E ratio</th>
+            <th>Div yield</th>
+            <th>CDP score</th>
+            <th>52-wk high</th>
+            <th>52-wk low</th>
+          </tr>>
+          </thead>>
+            <tbody>
+            <tr>
+              <td>88.90</td>
+              <td align ="right">90.32</td>
+              <td align ="right">87.59</td>
+              <td align ="right">1.15T</td>
+              <td align ="right">18.22</td>
+              <td align ="right">-</td>
+              <td align ="right">A-</td>
+              <td align ="right">151.55</td>
+              <td align ="right">83.34</td>
+            </tr>>
+            </tbody>
+          </table>
```

Comment:
Using a table is not a good choice to display this data. You're trying to display 9 fields within a single row within a small space.